parkingspaces/src/main/java/com/baspiotr/parkingspaces/domain/services/UserService.java

```diff
@@ -0,0 +1,54 @@
+package com.baspiotr.parkingspaces.domain.services;
+
+import com.baspiotr.parkingspaces.domain.model.Parking;
+import com.baspiotr.parkingspaces.domain.model.User;
+import com.baspiotr.parkingspaces.domain.repository.ParkingRepository;
+import com.baspiotr.parkingspaces.domain.repository.UserRepository;
+import com.baspiotr.parkingspaces.domain.utils.Fee;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class UserService {
+
+    private final ParkingRepository parkingRepository;
+    private final UserRepository userRepository;
+
+    //todo 1) As a driver, I want to start the parking meter, so I don’t have to pay the fine for the invalid parking.
+
+    public void startParkingMeter(int userId) {
+
+        User user = userRepository.findOne(userId);
+        Parking parking = parkingRepository.findAll().get(0);
```

Comment:
get(0) is very poor. Perphaps findFirst would be better if you say there is only 1 parking
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/
3.4.5. Limiting query results

Reply:
but you need to check nulls too, what it there is no parking? and handle exceptions